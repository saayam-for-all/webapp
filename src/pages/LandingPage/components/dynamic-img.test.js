import { render } from "@testing-library/react";
import Images from "./Dynamic_img";

describe("Images", () => {
  it("renders correctly", () => {
    const tree = render(<Images />);
    expect(tree).toMatchSnapshot();
  });
});
