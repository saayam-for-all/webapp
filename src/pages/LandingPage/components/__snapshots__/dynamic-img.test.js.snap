// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Images renders correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="images"
      >
        <div
          class="images-slide"
        >
          <img
            src="Employment.gif"
          />
          <img
            src="Matrimonial.gif"
          />
          <img
            src="Stocks.gif"
          />
          <img
            src="education.gif"
          />
          <img
            src="Finance.gif"
          />
          <img
            src="School.gif"
          />
          <img
            src="clothes.gif"
          />
          <img
            src="housing.gif"
          />
          <img
            src="Food.gif"
          />
          <img
            src="Shopping.gif"
          />
          <img
            src="college_admissions.gif"
          />
          <img
            src="Gardening.gif"
          />
          <img
            src="Sports.gif"
          />
          <img
            src="cooking.gif"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="images"
    >
      <div
        class="images-slide"
      >
        <img
          src="Employment.gif"
        />
        <img
          src="Matrimonial.gif"
        />
        <img
          src="Stocks.gif"
        />
        <img
          src="education.gif"
        />
        <img
          src="Finance.gif"
        />
        <img
          src="School.gif"
        />
        <img
          src="clothes.gif"
        />
        <img
          src="housing.gif"
        />
        <img
          src="Food.gif"
        />
        <img
          src="Shopping.gif"
        />
        <img
          src="college_admissions.gif"
        />
        <img
          src="Gardening.gif"
        />
        <img
          src="Sports.gif"
        />
        <img
          src="cooking.gif"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
